doc-390 (#961)
spectrocloud/librarium: content/docs/04.5-devx/03-cluster-groups.md

* The Cluster Groups launched in the [developer's enterprise datacenter/environment](/devx/cluster-groups#tenantscope).

# Cluster Groups

> Cluster Group is an aggregation of host clusters deployed through Palette Tenant Admin console. Palette sandbox clusters can be launched into these host cluster groups. The cluster groups are created under two scopes:

Reviewer: ```suggestion
A Cluster Group is an aggregation of host clusters deployed through the Palette Tenant Admin console. Palette virtual clusters can be launched into host cluster groups. The cluster groups are created under two scopes:
```

Once the cluster group is created, the day two operations on them can be done as follows:

> 1. Select the `Cluster Group` to be managed and click `Settings` to access different cluster management activities. 

Reviewer: ```suggestion
1. Select the Palette Cluster Group to be managed and click **Settings** to access different cluster management activities. 
```
